src/Components/sharedComponents/BetaLabel.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { Label } from '@patternfly/react-core';
+
+import './BetaLabel.scss';
+
+const BetaLabel = () => {
+  return <Label className="beta-label">Beta</Label>;
```

Comment:
This doesn't seem to pick up the `background-color` correctly... would it accomplish the same thing if we just passed in the `color="green"` prop? That results in:
![image](https://user-images.githubusercontent.com/95542540/231733166-5d81bcca-a73d-4f8d-b28b-dea307463079.png)

Reply:
The colour is still different than the designs. And outside of the popover the sass works fine, inside not. No idea why ._.

Reply:
Hmm... I didn't check the designs to be honest, so I didn't notice it was the wrong green. You're right it is quite a bit darker. I guess other apps on Insights will use this same design so it is probably important to be consistent.